### ISSUE TITLE
Add a nuance in wording for not generalizing the cognitive group

### ISSUE THREAD

**Issue by jake-abma** [w3c/wcag#329]
https://www.w3.org/WAI/WCAG21/Understanding/identify-input-purpose

> This allows for additional machine-processing for personalization and other automation, such as identifying and applying familiar terms or symbols to the inputs, which are needed for users with cognitive disabilities to be able to use the web.

The last part of the sentence is very generalizing for the cognitive group, the:

> which are needed for users with cognitive disabilities to be able to use the web

The biggest group of people with disabilities is the cognitive group and  we should make a nuance as not all people in this group need this. Although I don't have the metrics probably most of them will not need it necessarily to be able to use the web.

**Comment by jake-abma** [comment]
I see a proper description at the Benefit section, maybe use that one or part of it?!:

> Benefits
> Following the success criterion benefits users with various cognitive disabilities including people with language and memory related disabilities, and disabilities that affects executive function and decision making.

**Comment by jake-abma** [comment]
I see the text has changed since, but this remark still stands. The text now reads:

> The ability to programmatically declare the specific kind of data expected in a particular field makes filling out forms easier, **especially for people with cognitive disabilities.**

The nuance should be added in the form of " including people with language and memory related disabilities, and disabilities that affects executive function and decision making."

**Comment by alastc** [comment]
Can we keep it simpler? E.g:

> which are needed for users with **certain** cognitive disabilities to be able to use the web.

**Comment by jake-abma** [comment]
Update for 2.3?

**Comment by patrickhlauke** [comment]
https://github.com/w3c/wcag/pull/1817
